pubspec.yaml

```diff
@@ -43,3 +43,7 @@ flutter:
 
   uses-material-design: true
 
+  assets:
```

Comment:
this can also be removed now once you prefer to utilize the existing/built-in icons based on my feedback above 🙂